maintainers/maintainer-list.nix

```diff
@@ -15939,4 +15939,11 @@
     github = "wuyoli";
     githubId = 104238274;
   };
+  sielicki = {
```

Comment:
Please keep the list alphabetically sorted.

Reply:
Fixed, thanks.